src/relay/backend/build_module.cc

```diff
@@ -101,7 +139,18 @@ struct GraphCodegen {
     return ret;
   }
 
+  runtime::AOTMetadata GetAOTMetadata() {
```

Comment:
If the relevant information is available in LoweredOutput of AoTExecutorCodegen, we should not need to get this one seperately -- also look at my comment where this information is passed onto CreateMetadataModule(...)